docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '3.3'
+
+services:
+  bot:
+    image: "${BOT_IMAGE_NAME}"
+    container_name: "${BOT_CONTAINER_NAME}"
```

Comment:
why not use automatic naming?

Reply:
*  ${VARIABLE:-default} evaluates to default if VARIABLE is unset or empty in the environment.
*    ${VARIABLE-default} evaluates to default only if VARIABLE is unset in the environment.

should be better